src/utils/styles/main.scss

```diff
@@ -0,0 +1,16 @@
+* {
```

Comment:
move to all styles to "src/assets/styles/main.scss"

Reply:
Do you also mean components` styles?